tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -504,6 +507,11 @@ property has been added to our interface. You can now change this property and
 when you run your project, you will see that our Godot icon travels along a
 larger figure.
 
+.. note::
+
+    The hot reloading feature only works of you compiled your library with debug mode (default).
+    Also make sure it's enabled in the .gdextension file as shown above.
+
```

Comment:
This seem to be doubled with the changes above. I would rather specify the section in the ``gdextension file`` explanation of the ``[configuration]`` section